src/index.css

```diff
@@ -0,0 +1,23 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
```

Comment:
- [ ] Good job including one of the fonts, but kindly ensure all fonts used in the design are used in the implementation. These fonts are available on Google Fonts.